src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        Reader reader = new Reader();
+        Writer writer = new Writer();
+        writer.createCsvFile(toFileName);
```

Comment:
Creating a CSV file before writing to it is unnecessary if you're using 'FileWriter' or similar, as it will create the file if it doesn't exist when opening the stream.